sdk/scheduler/src/main/java/com/mesosphere/sdk/specification/yaml/RawVip.java

```diff
@@ -24,4 +30,12 @@ public int getPort() {
     public String getPrefix() {
         return prefix;
     }
+
+    public String getProtocol() {
+        return protocol;
+    }
+
+    public Boolean isAdvertise() {
```

Comment:
It's not a serialization requirement, I'd just been naming things in `Raw*.java` ala java beans, with the philosophy being to have as little logic in `Raw*.java` as possible.

Done. Went with `isAdvertised`.